core/tests/test_utils.py

```diff
@@ -43,7 +43,9 @@
 from google.appengine.api import apiproxy_stub_map
 from google.appengine.api import mail
 
+#Model for Oppia explorations for valid model names
 (exp_models,) = models.Registry.import_models([models.NAMES.exploration])
+#Platform-specific current_user_services module
```

Comment:
For line no. 46 and 48: Please add a space between '#' and first word. Also add a period (.) at the end of the sentence.